Views/backend/nosto_tagging/model/account.js

```diff
@@ -37,13 +37,13 @@ Ext.define('Shopware.apps.NostoTagging.model.Account', {
     extend: 'Ext.data.Model',
     idProperty: 'id',
     fields: [
-        { name: 'id', type: 'int' },
-        { name: 'name', type: 'string' },
-        { name: 'url', type: 'string' },
-        { name: 'email', type: 'string' },
-        { name: 'shopId', type: 'int' },
-        { name: 'shopName', type: 'string' },
-        { name: 'details', type: 'string' }
+        {name: 'id', type: 'int'},
+        {name: 'name', type: 'string'},
+        {name: 'url', type: 'string'},
+        {name: 'email', type: 'string'},
+        {name: 'shopId', type: 'int'},
+        {name: 'shopName', type: 'string'},
+        {name: 'details', type: 'string'}
     ],
```

Comment:
This change will break the store admin. There must be space after the opening brace and before the closing brace